### PR TITLE
Create Whats new for March

### PR DESCRIPTION
**Pull request by BillWagner**
Run the What's new script for March 2021.

[Internal review link](https://review.docs.microsoft.com/en-us/dotnet/whats-new/2021-03?branch=pr-en-us-23587)

### CODE REVIEW

File: docs/whats-new/2021-03.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: ".NET docs: What's new for March 1 2021"
```
Comment (tdykstra) [inline docs/whats-new/2021-03.md:2]:
```suggestion
title: ".NET docs: What's new for March 2021"
```

File: docs/whats-new/2021-03.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+description: "What's new in the .NET docs for March 1, 2021 - April 1, 2021."
```
Comment (tdykstra) [inline docs/whats-new/2021-03.md:3]: Should we change the end date?

```suggestion
description: "What's new in the .NET docs for March 1, 2021 - March 31, 2021."
```

**Review by tdykstra — APPROVED** [review]
LGTM, a couple of suggestions on the dates.

**Review by gewarren — APPROVED** [review]
I think we should consider leaving the commit messages off of the "new articles" - they often duplicate the title of the article, and since it's a new article, the title should tell it all anyway.

**Comment by BillWagner** [comment]
I updated this per @gewarren 's suggestion. I like the cleaner look as well.

I'll merge later today, unless others have different feedback.

**Pull request merged**